clientInit.js

```diff
@@ -0,0 +1,40 @@
+const __controllerTemplate = 'Controller';
+const __appVersion = __ver || 0;
+
+var controller = typeof(controller) === 'undefined' ? null : controller;
+var firstTimeLoad = typeof(firstTimeLoad) === 'undefined';
+
+// Establish a new session whether first time or because of expiry / server restart
+function __bindController (newController, sessionExpiration) {
+    if (controller && typeof(controller.shutdown) === 'function') {
+        controller.shutdown();
+    }
+    controller = newController;
+    if (firstTimeLoad) {
```

Comment:
Do you need this `if`?  Can't you always just set it to false?

Reply:
I'm actually not sure why @lysandros even made this variable firstTimeLoad. It doesn't actually do _anything_. It's just initiliazed to true in this file and then switched to false in bindController.

Reply:
It made sense sometime ago when controller.refresh() was the else {} part. 
We can remove the variable and the check.